<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1101</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Video Stream</string>
     </attribute>
     <widget class="QVideoWidget" name="container" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>821</width>
        <height>491</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="toggleButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>86</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listCamera">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>10</y>
        <width>231</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="camName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera Identifier :</string>
      </property>
     </widget>
     <widget class="QLabel" name="distance">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Distance :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QListWidget" name="settingWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>881</width>
        <height>481</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>10</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>50</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Item</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>90</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="applyButton">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>230</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Note that Camera Name and IP Address is delimited by &amp;quot;\&amp;quot;: &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Formatting Camera Item. Example &amp;lt;NameOfCamera&amp;gt;\&amp;lt;IPAddress&amp;gt;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>QVideoWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
